helloapache/README.md

```diff
@@ -1,73 +1,72 @@
-
-## Option 2: Unattended
+#### Deploying with Default Parameters
 
-1. Create the file `answers.conf` with these contents:
+##### Using Atomic CLI
 
-    This sets up the values for the two configurable parameters (image and hostport) and indicates that kubernetes should be the orchestration provider.
-
-        [general]
-        provider = kubernetes
-
-        [helloapache-app]
-        image = centos/httpd # optional: choose a different image
-        hostport = 80        # optional: choose a different port to expose
-1. Run the application from the current working directory
+```sh
+sudo atomic run projectatomic/helloapache
+```
 
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
+##### Using Atomic App
 
+```sh
+sudo atomicapp run projectatomic/helloapache
+```
 
-1. As an additional experiment, remove the kubernetes pod and change the provider to 'docker' and re-run the application to see it get deployed on base docker.
+#### Deploying with User-Provided Parameters
 
-## Option 3: Install and Run
+##### Using Atomic CLI
 
-You may want to download the application, review the configuraton and parameters as specified in the Nulecule file, and edit the answerfile before running the application.
+```sh
+sudo atomic run projectatomic/helloapache --mode fetch --destination helloapache
+cd helloapache
+sudo atomic run projectatomic/helloapache --mode genanswers answers .
+sudo atomic run projectatomic/helloapache -a answers.conf .
+```
 
-1. Download the application files using `atomic run IMAGE --mode fetch`
+##### Using Atomic App
 
-        [sudo] atomic run projectatomic/helloapache --mode fetch
+```sh
+atomicapp fetch projectatomic/helloapache --destination helloapache
+cd helloapache
+atomicapp genanswers .
+atomicapp run -a answers.conf .
+```
 
-1. Rename `answers.conf.sample`
+### Interaction
 
-        mv answers.conf.sample answers.conf
+#### Using the Docker Provider
 
-1. Edit `answers.conf`, review files if desired and then run
+The default port is :80.
 
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
+```sh
+curl localhost
+<html-foobar>
+```
 
-## Test
-Any of these approaches should create a kubernetes pod or a running docker container. 
+#### Using the Kubernetes Provider
 
-With a kubernetes pod, once its state is "Running" curl the minion it's running on.
+By default kubernetes will assign an available external IP.
 
-```
-$ kubectl get pod helloapache
-POD                IP                  CONTAINER(S)       IMAGE(S)           HOST                LABELS              STATUS
-helloapache        172.17.0.8          helloapache        centos/httpd       10.3.9.216/         name=helloapache   Running
-$ curl 10.3.9.216
-<bunches_of_html_goodness>
-```
+```sh
+kubectl describe pod helloapache
 
-If you test the docker provider, once the container is running, curl the port on your localhost.
+▶ kubectl describe po helloapache
```

Comment:
s/po/pod/ - I know po works but it looks like a mistake to someone just reading along.